navbar/navbar.css

```diff
@@ -0,0 +1,51 @@
+@import url("https://fonts.googleapis.com/css2?family=Qwigley&display=swap");
+
+.nav-bar {
+	padding: 0.5rem 20rem;
```

Comment:
why huge padding?

Reply:
to give space from left and right.

Reply:
my suggestion would be make the navbar flex and justify-content:space-around so that you would get space on both edges , so that you don't have to write a media query only for resizing the padding for this

Reply:
ok will try this